packages/e2e-tests/puppeteer/__tests__/rps.test.ts

```diff
@@ -18,25 +14,36 @@ const HEADLESS = getEnvBool('HEADLESS');
 
 let browserA: Browser;
 let browserB: Browser;
+let metamaskA: Dappeteer;
+let metamaskB: Dappeteer;
 let rpsTabA: Page;
 let rpsTabB: Page;
 
 describe('completes game 1 (challenge by A, challenge by B, resign by B) and begins game 2 ', () => {
   beforeAll(async () => {
-    browserA = await setUpBrowser(HEADLESS, 100); // 100ms sloMo avoids some undiagnosed race conditions
-    browserB = await setUpBrowser(HEADLESS, 100); // 100ms sloMo avoids some undiagnosed race conditions. TODO: remove sloMo and address underlying problem
+    const browserPromiseA = setUpBrowser(HEADLESS, 0);
+    const browserPromiseB = setUpBrowser(HEADLESS, 0);
+
+    ({browser: browserA, metamask: metamaskA} = await browserPromiseA);
```

Comment:
Why the brackets?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assignment_without_declaration